modules/theme-tailwind/molecules/ProductBundleOptions.vue

```diff
@@ -84,6 +85,26 @@ defineProps({
 const selectedOptions = ref({} as any)
 const selectedLabel = ref({} as any)
 
+const updateFormValue = computed(() => {
+  if (isEmpty(product.bundle)) return
+
+  const newValue = {} as any
+  const ids = Object.keys(product.bundle)
+
+  for (const idGroup of ids) {
+    for (const idValue of Object.keys(product.bundle[idGroup])) {
+      const groupKey = idGroup + '-group'
+      const keyInModal = idGroup + '-group' + idValue + '-inModal'
+      const keyOutOfModal = idGroup + '-' + idValue + '-outOfModal'
```

Comment:
Why the split into outOfModal/inModal? Arent they same? And if not, shouldnt they be?